src/components/Request/Pay/Views/GeneralViews/AlreadyPaid.view.tsx

```diff
@@ -21,7 +21,7 @@ export const AlreadyPaidLinkView = ({ requestLinkData }: { requestLinkData: _con
     return (
         <Card className="shadow-none sm:shadow-primary-4">
             <Card.Header className="text-center">
-                <Card.Title>Sorry, this link has already been paid.</Card.Title>
+                <Card.Title>This link previously contained:</Card.Title>
```

Comment:
In `main` this line is like this:

https://github.com/peanutprotocol/peanut-ui/blob/6c6cc0484e07ca3fd562c90fa6815b07b7ef61cd/src/components/Request/Pay/Views/GeneralViews/AlreadyPaid.view.tsx#L29

`            <label className="text-h2">Payment Receipt</label>`

We should keep that and not overwrite!

Reply:
done